src/main/java/org/openRealmOfStars/player/fleet/Fleet.java

```diff
@@ -207,14 +207,14 @@ public int getY() {
    * @param pos Fleet's coordinate
    */
   public void setPos(final Coordinate pos) {
-    this.coordinate = pos;
+    this.coordinate = new Coordinate(pos);
```

Comment:
The answer is that. If you pass the coordinate from an other object that is the same object.
Eg.: Create a fleet at a planet. You pass the planet coordinate to the fleet. When the fleet moves (Coordinate is changed) the planet's coordinate is also changed. (See the FleetTest and PlanetTest, to understand what I mean.)